### PR TITLE
Quick tasks

### PR DESCRIPTION
**Pull request by brgruber92**
[IN:10281] updates the QuickTasks docs to reflect the most recent changes

### CODE REVIEW

File: docs/SpiraApps/QuickTasks.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Setup
+To effectively implement Quick Tasks in your product, you have to set up the Task Type that will be used and make it use a specific workflow. 
+
+### Product Template Setup
+You need to configure each product template for the product(s) you want to use Quick Tasks:
+
+1. Go to Tasks > Workflow and add a new Workflow called "Quick Task Workflow". Do not change anything else about this workflow.
+
+2. Go to Tasks > Type and add a new Type called "Quick Task" (or other name of your choice) and select the Workflow you just created
```
Comment (simonhbor) [inline docs/SpiraApps/QuickTasks.md:15]: First, delete empty lines between list items - not required. Empty lines are only required between paragraphs in markdown.

Second, is this required or optional? I think again it is optional

File: docs/SpiraApps/QuickTasks.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+![A row in the Task Type table containing the following: Type ID number, Type name, selected workflow, "No" (for pull request switcher), an empty radio button (for default type), "Yes" (for Active switcher). There is a red outlined box around the Type ID number and the selected workflow](img/quicktasks-task-type-setting.png)
+
+!!! info "Reusing a Task Type"
+    If your team already uses a special Task Type for miscellaneous to-do items, you can use that instead of creating a new one, just make sure to associate it with the Quick Task Workflow you created on step 1.
+
+### Product Settings
+For each product that uses Quick Tasks, a product administrator must select the default Quick Task Type in the Quick Tasks SpiraApp product settings.
```
Comment (simonhbor) [inline docs/SpiraApps/QuickTasks.md:23]: What happens if they do not? Is this definitely required for the app to function?

File: docs/SpiraApps/QuickTasks.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+![SpiraApp product settings page with the Quick Task Type ID Number outlined by a red box](img/quicktasks-product-settings.png)
+
+## Using the SpiraApp
+
+!!! warning "Users must have 'Modify All' Tasks permission in order to complete their owned tasks using this widget"
+!!! warning "Users must have 'View' permissions for Releases in order to use this widget"
```
Comment (simonhbor) [inline docs/SpiraApps/QuickTasks.md:30]: put an empty line before this

**Review by simonhbor — CHANGES_REQUESTED** [review]
Some questions to help improve the clarity of how the app works and what is needed vs optional / recommended. Otherwise good - I will make more a marketing intro to explain the benefits of using the app on merge. If you wish to do this please feel free of course :)

File: docs/SpiraApps/QuickTasks.md (hunk 2 above)
Comment (brgruber92) [reply docs/SpiraApps/QuickTasks.md:23]: Yes, and there's an error message if nothing was selected

**Comment by brgruber92** [comment]
PR changes done

**Pull request merged**